lib/rules/require-prop-comment.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @author *****your name*****
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+
+// ...
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require prop should have a comment',
+      categories: undefined,
+      url: 'https://eslint.vuejs.org/rules/require-prop-comment.html'
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      // ...
+    }
+  },
+  /** @param {RuleContext} context */
+  create(context) {
+    const sourceCode = context.getSourceCode()
+
+    /**
+     * @param {import('../utils').ComponentProp[]} props
+     */
+    function verifyProps(props) {
+      for (const prop of props) {
+        if (!prop.propName) {
+          continue
+        }
+        const beforeComments = sourceCode.getCommentsBefore(prop.node)
+
+        if (beforeComments.length > 0) continue
+        context.report({
+          node: prop.node,
+          message: 'The "{{name}}" property should have a comment.',
+          data: {
+            name: prop.propName
+          }
+        })
+      }
+    }
+    // 合成选择器
```

Comment:
Please only write English comments (even if it is bad English), then all maintainers have a chance to understand this.

Reply:
The above problems will be solved in the near future